src/pages/HistoricalStatistics/BlockchainSize/index.tsx

```diff
@@ -0,0 +1,63 @@
+// react
+import { useEffect, useState } from 'react';
+// third party
+import { Skeleton } from '@material-ui/lab';
+// application
+import { Container } from '@pages/HistoricalStatistics/StatisticsOvertime.styles';
+import * as URLS from '@utils/constants/urls';
+import { useFetch } from '@utils/helpers/useFetch/useFetch';
+import { PeriodTypes, transformBlockchainSize } from '@utils/helpers/statisticsLib';
+import { CHART_DEFAULT_PERIOD, periods, info } from '@utils/constants/statistics';
+import { useBackgroundChart } from '@utils/hooks';
+import { TLineChartData, TTransactionsChart } from '@utils/types/IStatistics';
+import { EChartsLineChart } from '../Chart/EChartsLineChart';
+
+function BlockchainSize() {
+  const [chartData, setChartData] = useState<TLineChartData | null>(null);
+  const [currentBgColor, handleBgColorChange] = useBackgroundChart();
+  const [period, setPeriod] = useState<PeriodTypes>(CHART_DEFAULT_PERIOD);
+  const fetchStats = useFetch<{ data: Array<TTransactionsChart> }>({
+    method: 'get',
+    url: URLS.GET_BLOCKS_CHARTS,
+  });
+  useEffect(() => {
+    const loadLineChartData = async () => {
+      const data = await fetchStats.fetchData({
+        params: { period, sortDirection: 'DESC', sqlQuery: 'SUM(size)' },
```

Comment:
`sqlQuery` here should be changed to not pass sql directly. Maybe such parameter will fit: `aggregate: 'sumSize'`